src/main/edu/colostate/cs/cs414/method_men/jungle/client/gui/LoginPage.java

```diff
@@ -0,0 +1,111 @@
+package edu.colostate.cs.cs414.method_men.jungle.client.gui;
+
+import javax.swing.*;
+import javax.swing.border.LineBorder;
+import java.awt.*;
```

Comment:
IntelliJ is telling me that `javax.swing.border.LineBorder` and `java.awt.*` are never used. I think we should get rid of them.